tests/features/accounts.go

```diff
@@ -48,47 +46,42 @@ func parseAccountsDataTable(accountsDataTable *gherkin.DataTable) ([]*AccountEnt
 }
 
 func (ctx *Context) IHaveTheFollowingAccounts(accountsDataTable *gherkin.DataTable) error {
-	accounts, err := parseAccountsDataTable(accountsDataTable)
+	accountsData, err := parseAccountsDataTable(accountsDataTable)
 	if err != nil {
 		return err
 	}
 
 	// Create an archive node for each account and send them funds
-	for _, account := range accounts {
-		nodeName, err := ctx.cluster.RunArchiveNode()
+	for _, accountData := range accountsData {
+		account, err := ctx.accountsStorage.NewAccount("test")
 		if err != nil {
 			return err
 		}
 
-		res, err := ctx.cluster.Exec(nodeName, `eth.coinbase`)
-		if err != nil {
-			return err
-		}
-		coinbaseQuotes := res.StdOut
+		ctx.accounts[accountData.AccountName] = account
 
-		ctx.accountsNodeNames[account.AccountName] = nodeName
-		ctx.accountsCoinbase[account.AccountName] = strings.TrimSpace(strings.Replace(coinbaseQuotes, `"`, "", 2))
-
-		res, err = ctx.cluster.Exec(ctx.genesisValidatorName,
+		// TODO: Use genesis validator key to send a transaction using the client instead of Exec
```

Comment:
move this information into a ticket? TODOs should not go in to stable branches

Reply:
Done